src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -0,0 +1,164 @@
+import { Box, color, Flex, Sans, Serif } from "@artsy/palette"
+import { ArtworkTombstone_artwork } from "__generated__/ArtworkTombstone_artwork.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { NativeModules, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+const Constants = NativeModules.ARCocoaConstantsModule
+
+type Artist = ArtworkTombstone_artwork["artists"][0]
+
+export interface ArtworkTombstoneProps {
+  artwork: ArtworkTombstone_artwork
+}
+
+export interface ArtworkTombstoneState {
+  showingMoreArtists: boolean
+}
+
+export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, ArtworkTombstoneState> {
+  state = {
+    showingMoreArtists: false,
+  }
+  handleTap(href) {
+    SwitchBoard.presentNavigationViewController(this, href)
+  }
+
+  showMoreArtists = () => {
+    this.setState({
+      showingMoreArtists: !this.state.showingMoreArtists,
+    })
+  }
+
+  renderSingleArtist(artist: Artist) {
+    return (
+      <React.Fragment>
+        <Flex flexDirection="row">
+          {this.renderArtistName(artist.name, artist.href)}
+          <Sans color={color("black60")} size="6" mx={1}>
+            &middot;
+          </Sans>
+          <Sans color={color("black60")} size="4">
+            Follow
+          </Sans>
+        </Flex>
+      </React.Fragment>
+    )
+  }
+
+  renderArtistName(artistName, href) {
+    return href ? (
+      <>
+        <TouchableWithoutFeedback onPress={this.handleTap.bind(this, href)}>
+          <Serif size="5t" weight="semibold">
+            {artistName}
+          </Serif>
+        </TouchableWithoutFeedback>
+      </>
+    ) : (
+      <>
+        <Serif size="5t" weight="semibold">
+          {artistName}
+        </Serif>
+      </>
+    )
+  }
+
+  renderMultipleArtists = () => {
+    const {
+      artwork: { artists },
+    } = this.props
+    return artists.map((artist, index) => {
+      const isHidden = index > 2 && !this.state.showingMoreArtists
```

Comment:
I wonder whether it would be better if we were to use `this.state.showingMoreArtists` to slice the first three elements out of the `artists` array rather than mapping over the whole thing every time and dealing with hiding extraneous artists in here?